projects/elements/src/lib/lazy-elements/lazy-elements.injectors.ts

```diff
@@ -0,0 +1,6 @@
+import { InjectionToken } from '@angular/core';
+import { LazyElementsLoaderService } from './lazy-elements-loader.service';
+
+export const ELEMENTS_CONFIG = new InjectionToken<LazyElementsLoaderService>(
```

Comment:
Can be defined inside of the module to keep it simpler, and the type is NOT `<LazyElementsLoaderService>` but `<ElementConfig[]>`